packages/frontend-2/lib/core/configs/apollo.ts

```diff
@@ -334,7 +334,7 @@ function createWsClient(params: {
 
 const coreShouldSkipLoggingErrors = (err: ErrorResponse): boolean => {
   // These fields have special auth requirements and will often throw errors that we don't want to log
-  const specialAuthFields = ['invitedTeam', 'billing', 'domains']
+  const specialAuthFields = ['invitedTeam', 'billing', 'domains', 'subscription']
```

Comment:
@Mikehrn We have errorPolicy: 'all' so its fine that the Workspace.subscription field errors out and returns null for non-admins, but we should also remember to update this array (or the skipLoggingErrors param of the specific query) to ensure we don't log these as errors in the console and scare devs (which is how I was told to investigate this hehe)